tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTokenInterceptor.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.pipeline.tesadapter.service;
```

Comment:
I believe it would be loks better in  `controller` package, since it registered for controller

Reply:
fixed